share/revng/rcc-config.yml

```diff
@@ -131,6 +130,7 @@ read_passes:
               - "<="
               - '//\s*WIP'
               - '#if\s*[01]'
+              - "#pragma clang optimize off"
```

Comment:
Why do we need this?

Reply:
This might get committed :-1:.
I know it's not the best but you can have a command that does `orc shell -c revng ctest -j16 && orc install test/revng` if you want to bypass `check-conventions`. Usually I don't run `ctest` an `test/revng` though, I'm usually interested only in one of the two.

Reply:
Gotcha.

Reply:
Dropped.